CorsixTH/Lua/app.lua

```diff
@@ -1531,15 +1535,26 @@ end
 
 --! Function to check the loaded game is compatible with the program
 --!param save_version (num)
+--!param demo_flag (bool)
 --!return true if compatible, otherwise false
-function App:checkCompatibility(save_version)
+function App:checkCompatibility(save_version, demo_flag)
```

Comment:
`demo_flag` isn't clear wrt what thing it represents, the graphics set or the save game.

Reply:
Specifically, neither. But as per your general comment I'll look into graphics set